Components/CreateAccount/CreateAccount.js

```diff
@@ -102,7 +108,11 @@ export class CreateAccount extends Component {
 		try {
 			const response = await fetch('https://sophia-be.herokuapp.com/api/v1/clients/', options);
 			const account = await response.json();
-      await this.logInClient(account.username, account.password)
+			// await logInClient(account.username, account.password)
+			// console.log(user)
+			// this.props.logIn(user)
+				this.setState(initialState);
+    	this.props.navigation.navigate('ClientProfile');
```

Comment:
@kalex19 , 

This 'navigate' takes the user directly to ClientProfile right after he/she/they log in? Shouldn't it take them to the 'Homepage', where they get to pick whether they want to look at their profile or their lists?

Reply:
Yep, it goes to User now.